terraform/neutron.tf

```diff
@@ -65,14 +65,10 @@ resource "openstack_networking_subnet_v2" "subnet_mgmt" {
   }
 }
 
-data "openstack_networking_network_v2" "external" {
-  name = var.external != "" ? var.external : data.openstack_networking_network_v2.extnet.name
-}
-
```

Comment:
I like the idea of avoiding multiple reads for the same OS resource. 
But I also prefer to put data sources next to the resources that reference them or have a shared `data.tf` file where you can put shared data.  

Overall I do not have a strong preference here, it is just a suggestion on how you could potentially improve script readability.

Reply:
See b15bb701f0c19e94504f2971aa19dd92c1aebde2